WhateverGreen/kern_igfx.cpp

```diff
@@ -1072,8 +1072,32 @@ bool IGFX::wrapAcceleratorStart(IOService *that, IOService *provider) {
 		that->setName("IntelAccelerator");
 	
 	if (callbackIGFX->forceSKLAsKBL) {
-		DBGLOG("igfx", "disabling VP9 hw decode support on Skylake when using KBL kexts");
+		DBGLOG("igfx", "disabling VP9 hw decode support on Skylake with KBL kexts");
 		that->removeProperty("IOGVAXDecode");
+
+		if (auto hevcDecodeCap = OSDynamicCast(OSDictionary, that->getProperty("IOGVAHEVCDecodeCapabilities"))) {
```

Comment:
That's C++17, this feature is cool, but it does not make the code more readable when you make so many nested levels. Could you please move it to a new function and instead return from it on failing to get the property or allocate a dictionary?

Reply:
https://github.com/acidanthera/WhateverGreen/commit/a9edfe199550a37c0f1172f836a9d18dd09919eb